src/helpers/skins.js

```diff
@@ -0,0 +1,9 @@
+const requireFile = require.context(
+    '@snapshot-labs/snapshot-spaces/skins/',
+    true,
+    /[\w-]+\.scss$/
+);
+  
+requireFile.keys().map(file => requireFile(file));
+  
+export default requireFile.keys().map(file => file.replace('./', '').replace('.scss', ''));
```

Comment:
this will load all the snapshot skins into the iframe so they can be reference and override the css variables var(--bg-color) etc - we took it straight from the snapshot repo and it is how they load all the skins into their own project.